app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+   License, v. 2.0. If a copy of the MPL was not distributed with this
+   file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home
+
+import android.content.Context
+import android.os.Build
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.annotation.RequiresApi
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.fragment_home.*
+
+import org.mozilla.fenix.R
+import android.widget.RelativeLayout
+import android.graphics.Color
+import androidx.recyclerview.widget.LinearLayoutManager
+
+
+class HomeFragment : Fragment() {
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_home, container, false)
+    }
+
+    private lateinit var sessionsAdapter: SessionsAdapter
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        sessionsAdapter = SessionsAdapter(requireContext())
+
+        toolbar_wrapper.clipToOutline = false
+        toolbar.apply {
+            textColor = ContextCompat.getColor(context, R.color.searchText)
+            textSize = 14f
+            hint = context.getString(R.string.search_hint)
+            hintColor = ContextCompat.getColor(context, R.color.searchText)
+        }
+
+        session_list.apply {
```

Comment:
We'll tease out the view UI logic later to avoid God fragments.

Reply:
/agree